src/electron/flux/action/window-state-actions.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Action } from 'common/flux/action';
+
+export class WindowStateActions {
+    public readonly setDeviceConnectRoute = new Action<void>();
```

Comment:
I'm wondering if this should be more generic, like `setRoute = new Action<RoutePayload>()`... not sure if it's a good idea to have a set of valid routes defined on a separated file, but it sounds like each "view" may have it's own configuration (like the route -which can be the id- and the default window size)

Reply:
RoutePayload  type => { routeId : ElectronRoute}

Reply:
updated in most recent commit